docs/using/queueing.rst

```diff
@@ -52,11 +52,12 @@ The following parameters are available for HTCondor:
   * ``kind``: This is the kind of virtual environment to use:
 
     * ``conda``
-    * ``virtualenv`` (not yet supported)
+    * ``venv``
     * ``local`` (no virtual environment)
 
   * ``name``: This is the name of the environment to use in case a virtual
-    environment is used.
+    environment is used. It should be the name when ``conda`` is used and
+    the absolute path to the virtualenv when ``venv`` is used.
```

Comment:
no support for a relative path? What is the main rationale behind this?

Reply:
The main question is: relative to what? The YAML? the CWD? We had this issue in #224, #127, #191.

If we can answer this, then we can also use a relative path.

Reply:
i agree. Absolute will keep it simple.

Reply:
I think relative paths are simpler, problem is always the same: determining the "relative to" to compute an absolute path and fix it in the yaml.

Reply:
We have been following the "relative to YAML" idea and I updated it to reflect that.